lib/repl.js

```diff
@@ -794,7 +794,16 @@ function complete(line, callback) {
     filter = match[1];
     var dir, files, f, name, base, ext, abs, subfiles, s;
     group = [];
-    var paths = module.paths.concat(require('module').globalPaths);
+    let paths = [];
+    if (completeOn === '.') {
+      group = ['./', '../'];
+    } else if (completeOn === '..') {
+      group = ['../'];
+    } else if (/^\.\.?\//.test(completeOn)) {
```

Comment:
In this context no, there isn't, both `./` and `../` just mean that we should start collecting completions starting from `process.cwd()`. The whole magic with `path.resolve()` is happening in the loop below.